### PR TITLE
Drop simple factory #2416

### PR DESCRIPTION
**Pull request by magiccookie**

### CODE REVIEW

File: apps/ehealth/test/data/address_type.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (AlexKovalevych) [inline apps/ehealth/test/data/address_type.json]: please use factory for address

File: apps/ehealth/test/support/address-factory.ex [hunk 2]
```diff
@@ -0,0 +1,24 @@
+defmodule EHealth.AddressFactory do
+  @moduledoc false
+
+  defmacro __using__(_opts) do
+    quote do
+      def address_factory do
+        %{
+          type: "REGISTRATION",
```
Comment (AlexKovalevych) [inline apps/ehealth/test/support/address-factory.ex]: you can use string keys and avoid extra conversion

File: apps/ehealth/test/support/address-factory.ex [hunk 3]
```diff
@@ -0,0 +1,24 @@
+defmodule EHealth.AddressFactory do
```
Comment (AlexKovalevych) [inline apps/ehealth/test/support/address-factory.ex]: please rename `address-factory.ex` to `address_factory.ex` and move it under `factories` directory

Comment (magiccookie) [reply apps/ehealth/test/support/address-factory.ex]: done

File: apps/ehealth/test/support/address-factory.ex (hunk 2 above)
Comment (magiccookie) [reply apps/ehealth/test/support/address-factory.ex]: done

File: apps/ehealth/test/data/address_type.json (hunk 1 above)
Comment (magiccookie) [reply apps/ehealth/test/data/address_type.json]: done

**Pull request merged**